docs/data/material/experimental-api/css-variables/css-variables.md

```diff
@@ -187,14 +187,18 @@ export default class MyDocument extends Document {
 
 #### Gatsby
 
-To use the API with a Gatsby project, add the following code to the custom [`gatsby-ssr.js`](https://www.gatsbyjs.com/docs/reference/config-files/gatsby-ssr/) file:
+To use the API with a Gatsby project, add the following code to the custom [`gatsby-ssr.jsx`](https://www.gatsbyjs.com/docs/reference/config-files/gatsby-ssr/) file:
```

Comment:
Does this need to change?

From the official site:
```md
To use the [Gatsby SSR APIs](https://www.gatsbyjs.com/docs/reference/config-files/gatsby-ssr/), create a file called gatsby-ssr.js in the root of your site
```

Reply:
I was mistaken and reverted.